ovs/extensions/api/client.py

```diff
@@ -170,10 +170,10 @@ def wait_for_task(self, task_id, timeout=None):
         """
         start = time.time()
         task_metadata = {'ready': False}
-        while task_metadata['ready'] is False:
+        while task_metadata['status'] not in ('FAILURE', 'SUCCESS'):
```

Comment:
The ```status``` key is not in ```task_metadata``` at this point. But because of the duplicate ```... not in ('FAILURE', 'SUCCESS')```, it might be nicer to have something like

````
finished = False
while finished is False:
    # do the actual call
    finished = task_metadata['status'] in ['FAILURE', 'SUCCESS']
    if finished is False:
        time.sleep(1)
```